### ISSUE TITLE
Investigate providing a GUI

### ISSUE THREAD

**Issue by mathieudutour** [mathieudutour/medium-to-own-blog#53]
I quite like the approach from https://github.com/alex-saunders/glicky

We could have `npx medium-to-own-blog` starts a local webserver and launch a webpage with a nice UI instead of doing everything in the terminal which might be a show stopper for some people

**Comment by dbarochiya** [comment]
The very strange idea came to my mind, what if we can have a dedicated website powered by netfliy functions which can generate the blog for users without touching the terminal itself.

This will open up the opportunity to the wide audience to own their blog, but we need to make sure these things are into place before even thinking about it.

* How to update the template as the codebase is getting updated
* Integrate the netlify CMS such that they can use native UI to create blogs
* Find a way to integrate the medium-like-editor with netlify CMS 

I know this sounds like a dream but it would be awesome if these things can take place

**Comment by mathieudutour** [comment]
I thought about that as well but but running the entire script on a website is a lot harder: you need to create a git repo on the server, push it with the credentials of the user, etc..

**Comment by paramaggarwal** [comment]
My opinion on this is that people who would not be ok running a script in the terminal for this would not be the target audience for this. They would be better off using Ghost Pro hosted plan.

The value of this tool is for people ready to dabble with GitHub+Netlify - while these are relatively easy tools, they still require some level of technical comfort. Adding a browser-based local UI for this tool doesn't seem like a useful benefit for such a task.

**Comment by dbarochiya** [comment]
I totally agree with you @paramaggarwal, even I mentioned this in one of the other discussions. But what I feel is if we can make a better free alternative for the people while keeping the entry barrier as low as possible for them to get in, it would be a great thing to achieve
